dapp/src/routes/Splash/splash.scss

```diff
@@ -1,11 +1,16 @@
-.splash-container{
+.splash-container {
     margin: 50px auto;
     max-width: 800px;
     display: flex;
     flex-direction: column;
     padding: 24px;
+
```

Comment:
Remove file and use Tailwind classes

Reply:
I've just remade scss to tailwind css classes, files have not deleted. Is it okay?
I may also clean html, add name of the classes and remove all tailwind classes in files, like these, to each component, if need.
May you look, please if it's okay.

Reply:
Great!